lib/main.js

```diff
@@ -0,0 +1,69 @@
+/** @babel */
+
+import { CompositeDisposable } from 'atom';
+import TerminalElement from './terminal-element';
+import TerminalSession from './terminal-session';
+import config from './config';
+
+const TERMINAL_TAB_URI = 'terminal-tab://';
+
+export default {
+
+  config,
```

Comment:
Interesting, does this expand the object into a property named `config`?  That's a nice trick

Reply:
Yep! Here's some good information on it: https://eslint.org/docs/rules/object-shorthand.

Reply:
Very cool, I've seen that in Go but didn't know that ES6 had it!